test-helpers/package.json

```diff
@@ -42,8 +42,8 @@
     "prettier": "^2.8.8",
     "serve-handler": "^6.1.3",
     "style-loader": "^2.0.0",
-    "vite": "^5.0.0 || ^6.0.0",
-    "vite-plugin-inspect": "^0.8.1",
+    "vite": "^5.0.0 || ^6.0.0 || ^7.0.0",
+    "vite-plugin-inspect": "^11.3.0",
```

Comment:
The `vite` version range is really only there to keep [`manypkg`](https://github.com/Thinkmill/manypkg) happy. I'm happy to stick with `^11.3.0` as we'll only actually use Vite@7 in the repo.